.circleci/config.yml

```diff
@@ -1853,8 +1853,7 @@ jobs:
       - setup-python-venv:
           executor: <<pipeline.parameters.machine-image>>
           extras-requires: anchorecli
-      - run: sudo snap install yq
-      - run: sudo apt-get install -y jq
+      - run: sudo snap install yq jq
```

Comment:
```suggestion
      - run: sudo snap install yq && sudo snap install jq
```

Reply:
At least locally for me, you can't pass multiple dependencies to snap and install them at once.

Reply:
strange: I tried on circleci with ssh job and it worked. oh well.